### DISCUSSION TITLE
Welcome to excel-streaming-reader Discussions!

### DISCUSSION THREAD

**Discussion by pjfanning** [pjfanning/excel-streaming-reader#136] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jdussouillez** [comment]
I'm not using it on a daily basis so I don't have any feature ideas or questions. But I just want to say: thank you ! The streaming reader works great and it's quite easy to set it up.
Also, I really appreciate the POI dependencies are updated regularly (so I don't have to wait months/years to be able to update POI in my projects).

**Reply by pjfanning** [reply]
Thanks @jdussouillez - I'm glad to hear the lib is working well for you.
